src/components/Person/Person.jsx

```diff
@@ -1 +1,23 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  return (
+    <>
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+      {age && (
+        <p className="Person__age">
+            {`I am ${age}`}
+        </p>
+      )}
+      <p className="Person__partner">
+        {isMarried === true ? `${partnerName} is my ${sex === 'm' ? 'wife' : 'husband'}` : 'I am not married'}
```

Comment:
This comment is not fixed. Feel free to ask questions in the chat if you need help.
![image](https://github.com/mate-academy/react_person/assets/99403198/9f2aebdd-0f35-4c5f-b3c6-18aff395b9f6)

You can create a variable `partner` for example
```suggestion
        {isMarried ? `${partnerName} is my ${partner}` : 'I am not married'}
```